packages/pyolite-kernel/src/kernel.ts

```diff
@@ -8,6 +8,9 @@ import { PromiseDelegate } from '@lumino/coreutils';
 
 import worker from './worker?raw';
 
+import widgetsnbextension from '../py/widgetsnbextension/dist/widgetsnbextension-3.5.0-py3-none-any.whl';
+import nbformat from '../py/nbformat/dist/nbformat-4.2.0-py3-none-any.whl';
+import ipykernel from '../py/ipykernel/dist/ipykernel-5.5.5-py3-none-any.whl';
```

Comment:
We're going to need a way to automate the _hell_ out of this, and make it build-time serviceable. I did a similar thing on [ipydrawio](https://github.com/deathbeds/ipydrawio/blob/master/packages/ipydrawio-webpack/lib/_static.js), but we'll actually be building no-foolin packages, so worth a thought.